pkg/init/cmd/service/runc.go

```diff
@@ -103,9 +136,35 @@ func runcInit(rootPath string) int {
 
 		cleanup(path)
 		_ = os.Remove(pidfile)
+
+		// dump the log file outputs to os.Stdout/os.Stderr
+		if _, err = stdout.Seek(0, 0); err != nil {
+			log.Printf("Error resetting stdout of onboot service %s to console: %v", name, err)
+		} else if _, err := io.Copy(os.Stdout, stdout); err != nil {
+			log.Printf("Error writing stdout of onboot service %s to console: %v", name, err)
+		}
+
+		if _, err = stderr.Seek(0, 0); err != nil {
+			log.Printf("Error resetting stderr of onboot service %s to console: %v", name, err)
+		} else if _, err := io.Copy(os.Stderr, stderr); err != nil {
+			log.Printf("Error writing stderr of onboot service %s to console: %v", name, err)
+		}
+
 	}
 
 	_ = os.RemoveAll(tmpdir)
 
+	// make sure the link exists from /var/log/onboot -> /run/log/onboot
+	if err := os.MkdirAll(varLogDir, 0755); err != nil {
+		log.Printf("Error creating secondary log directory %s: %v", varLogDir, err)
+	} else if _, err := os.Stat(varLogLink); os.IsNotExist(err) {
+		// create the link
+		if err := os.Symlink(logDir, varLogLink); err != nil {
+			log.Printf("Error creating symlink from %s to %s: %v", varLogLink, logDir, err)
+		}
+	} else {
```

Comment:
dont bother with the `else` case error message

Reply:
OK